lib/core/Core.ts

```diff
@@ -149,7 +138,7 @@ export default class Core {
    *   2. An encoded DID Document prefixed by the DID method name. e.g. 'did:sidetree:<encoded-DID-Document>'.
    */
   public async handleResolveRequest (didOrDidDocument: string): Promise<ResponseModel> {
-    const currentTime = this.approximateTime!;
+    const currentTime = this.blockchainClock.getApproximateTime()!;
```

Comment:
Optional nit: we can consider rename `getApproximateTime()` to just `getTime()`, 'coz arguably how time is calculated now is an internal detail to `BlockchainClock`.